reference/CloudPay-API-Specification.yaml

```diff
@@ -1788,6 +1789,12 @@ paths:
                   receipt:
                     purchaseIdentifier: shdbfhgdsvfdv1234
                   platform: roku
+              amazon-example:
+                value:
+                  receipt:
+                    purchaseIdentifier: lkjhgfdsapoiuytrewq1234
+                    userId: a-amazon-userId
```

Comment:
1. I think we'll need to update the schema to have an optional field `userId` if it is needed for Amazon IAP
2. *an-amazon-userId

Reply:
good spot, i have added